lib/api/createReadStream.js

```diff
@@ -33,6 +34,7 @@ module.exports = function createReadStream(path, options, cb) {
     var close = request.bind(undefined, 'close', file, connection);
 
     var stream = new Readable();
+    stream.fileSize = fileSize;
```

Comment:
Can you update `createReadStream` test to include this?

Reply:
Done, I needed to upgrade the mininum Node version because eslint was complaining about used functionality not available in older Node versions.

Reply:
What was the issue exactly?

Reply:
The issue was that the total filesize was not exposed (as far as I can tell from the code and while debugging) and I need that for several reasons, one being range request.

Reply:
I've pushed some fixes on `master`, can you please merge (or rebase) your branch with it, revert your change on ESLint and check whether you still have the issue?

Reply:
Sorry I was not aware that you meant the change of the minimum node version when you said "the issue" 😄. I've created a new PR.